react-intro/react-intro-quiz-notes.md

```diff
@@ -0,0 +1,49 @@
+# react-intro-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What is React?
+
+React is a library that allows developers to build websites and interfaces using components
+
+- What is a React component?
+
+a self-contained collection of HTML, CSS and JavaScript that provide all of the functionality necessary for something like a form
+
+- How do you mount a React app (root component) to the DOM?
+
+import export
+
+- What are some other popular frontend frameworks?
+
+Vite
```

Comment:
Vite is technically a bundler. It's not specifically a frontend frameworks. We are looking for answers like "Vue", "Svelte", etc..